### PR TITLE
upgrade: celestia pkgs and light client ver

### PR DESCRIPTION
**Pull request by 0xRampey**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,19 +2,17 @@ version: "3.8"
...
-    image: ghcr.io/celestiaorg/celestia-node:v0.14.0
-    user: "10001:10001"
+    image: ghcr.io/celestiaorg/celestia-node:v0.16.0-rc0
     environment:
       NODE_TYPE: ${NODE_TYPE} # Ensure these environment variables are set in your shell or a .env file
       P2P_NETWORK: ${NETWORK}
     volumes:
       - ${NODE_STORE_DIR}:/home/celestia
-    command: celestia ${NODE_TYPE} init --p2p.network ${NETWORK}
+    command: celestia ${NODE_TYPE} init --p2p.network ${NETWORK} --experimental-pruning
```
Comment (aashutoshrathi) [inline docker-compose.yml:11]: We still want `--experimental-pruning`?

**Review by aashutoshrathi — APPROVED** [review]
LGTM rest

File: docker-compose.yml (hunk 1 above)
Comment (0xRampey) [reply docker-compose.yml:11]: its on devnet and prod so if anything breaks while testing locally, we'll know

Comment (aashutoshrathi) [reply docker-compose.yml:11]: I was wondering if the memory spikes on the LC are linked to that

Comment (aashutoshrathi) [reply docker-compose.yml:11]: would be interesting if we can delegate flexibility of adding these flags to the user of `go-daash`

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**